tests/rustdoc-ui/lints/check.stderr

```diff
@@ -4,7 +4,6 @@ warning: missing documentation for the crate
 LL | / #![feature(rustdoc_missing_doc_code_examples)]
 LL | |
 LL | |
```

Comment:
why are there still empty lines above?

Reply:
They correspond to `//~ ERROR` annotations, which are removed from view, but that are not considered empty by the code.